adhocracy4/follows/templatetags/react_follows.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+from django.utils.html import format_html
+
+register = template.Library()
+
+
+@register.simple_tag()
+def react_follows(project):
+    return format_html(
+        '<span data-euth-widget="follows" data-project={project}></span>',
```

Comment:
This widget won't work embedded, because it is not triggered by inline scripts. Either we have to fix the global javascript hook for embedding issue or it needs to changed to something along those lines:

    return format_html(
           '<span id="{mountpoint}" data-project="{project_slug}"></span>'
           '<script>window.adhocracy4.renderFollow(document.getElementById("{mountpoint}"))</script>',
           project_slug=project.slug,
           mountpoint="follow-button-" + project.slug,
    )

Also you need to hookup this in [index.js](https://github.com/liqd/adhocracy4/blob/master/index.js).